### PR TITLE
[Doc] Add description of yaml

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
### PR types

Others 
### PR changes

Docs 
### Describe

1. 添加 yaml 的撰写教程
2. 优化几处文档

### CODE REVIEW

File: docs/zh/development.md [hunk 1]
```diff
@@ -602,23 +602,191 @@ solver = ppsci.solver.Solver(
...
+    lr_scheduler = ppsci.optimizer.lr_scheduler.Step(**cfg.TRAIN.lr_scheduler)()
+    ...
+    ```
+
+    这些参数在实验过程中随时可能作为变量而被手动调整，在调整过程中如何避免频繁修改源代码导致试验记录混乱、保障记录完整可追溯便是一大问题，因此 PaddleScience 提供了基于 hydra + omegaconf 的
+    配置文件管理系统来解决这一问题。
+
+    将已有的代码修改成配置文件控制的方式非常简单，只需要将必要的参数写到 `yaml` 文件中，然后通过 hydra 在程序运行时读取、解析该文件，通过其内容控制实验运行即可，以 `bracket` 案例为例，具体包含以下几个步骤。
+
+    1. 则需在代码文件 `bracket.py` 所在目录下新建 `conf` 文件夹，并在 `conf` 下新建与 `bracket.py` 同名的 `viv.yaml` 文件，如下所示。
```
Comment (zhiminzhang0830) [inline docs/zh/development.md:645]: 应该是是 viv.py 吧

File: docs/zh/development.md [hunk 2]
```diff
@@ -602,23 +602,191 @@ solver = ppsci.solver.Solver(
...
+
+        ``` sh hl_lines="3-4"
+        PaddleScience/examples/fsi/
+        ├── viv.py
+        └── conf
+            └── viv.yaml
+        ```
+
+    2. 将 `viv.py` 中必要的超参数按照其语义填写到 `viv.yaml` 的各个层级的配置中，如 `mode`、`output_dir`、`seed`、方程参数、文件路径等参数为通用参数，直接填写在一级层级；而模型结构参数、训练轮数等参数则只与模型、训练相关，只需分别填写在 `MODEL`、`TRAIN` 层级下即可（`EVAL` 层级同理）。
```
Comment (zhiminzhang0830) [inline docs/zh/development.md:654]: 这样更通顺些
```
如通用参数 `mode`、`output_dir`、`seed`、方程参数、文件路径等，直接填写在一级层级
```

Comment (HydrogenSulfate) [reply docs/zh/development.md]: 已修改

File: docs/zh/development.md (hunk 1 above)
Comment (HydrogenSulfate) [reply docs/zh/development.md]: 已修改

**Review by zhiminzhang0830 — APPROVED** [review]
LGTM
